src/mame/drivers/minitel_2_rpic.cpp

```diff
@@ -51,6 +51,7 @@
 
 #include "cpu/mcs51/mcs51.h"
 #include "machine/timer.h"
+#include "machine/i2cmem.h"
 #include "video/ef9345.h"
```

Comment:
Please keep includes sorted – when there are a lot of them, it can get out of hand.

Reply:
Fixed.